aws/resource_aws_emr_cluster_test.go

```diff
@@ -5465,6 +5596,32 @@ resource "aws_iam_role_policy_attachment" "emr-autoscaling-role" {
 `, r)
 }
 
+func testAccAWSEmrClusterConfigMasterInstanceGroupMarket(rName, Market string) string {
```

Comment:
Nit: Same here -- otherwise okay 👍 

```suggestion
func testAccAWSEmrClusterConfigMasterInstanceGroupMarket(rName, market string) string {
```